app/labor/random_gif.rb

```diff
@@ -13,6 +13,12 @@ def initialize(_action = "congratulations")
       "y8Mz1yj13s3kI" => { aspect_ratio: 0.750 },
       "111ebonMs90YLu" => { aspect_ratio: 0.750 },
       "Sk5uipPXyBjfW" => { aspect_ratio: 0.422 },
+      "l0K48FkLfeSCzRA4M" => { aspect_ratio: 0.573 },
+      "3o7qDRd1DlF7P2TP3O" => { aspect_ratio: 0.517 },
+      "26h0qt6UOumsbJkyI" => { aspect_ratio: 0.442 },
+      "3o7qDOzZAhOslfIgMM" => { aspect_ratio: 0.807 },
```

Comment:
I think this `aspect_ratio` should be 0.829.

Width: 480
Height: 398

Reply:
I agree with this. BTW @Glennmen did you know about GitHub's suggestion feature? 

![screen shot 2019-03-04 at 11 34 07 am](https://user-images.githubusercontent.com/17884966/53747525-74019c80-3e71-11e9-9b15-9e5b5bae165c.png)

Reply:
@Zhao-Andy I did read about it in the past but was looking for it and couldn't find it (and was lazy and didn't look it up)... My bad!

Reply:
All good, happy to share it :) I'm a big fan of it!